pkg/rancher-desktop/components/Preferences/BodyVirtualMachine.vue

```diff
@@ -99,6 +105,7 @@ export default Vue.extend({
       <component
         :is="`preferences-virtual-machine-${ activeTab }`"
         :preferences="preferences"
+        :select-tab="selectTab"
```

Comment:
Since the `selectTab` method operates directly on the tab name, we can update this to `@update:tab="commitPreferences"` and remove the `selectTab` method entirely.